go/lru/disk_lru.go

```diff
@@ -324,7 +327,10 @@ func (d *DiskLRU) Put(ctx context.Context, lctx libkb.LRUContext, key string, va
 	defer func() {
 		// Commit the index
 		if err == nil && index != nil && index.IsDirty() {
-			d.writeIndex(ctx, lctx, index, true)
+			err := d.writeIndex(ctx, lctx, index, true)
```

Comment:
we shouldn't have a new `err` variable in this case but return the error since we are force writing the index